### PR TITLE
Add geocoding function to utils.py

### PR DESCRIPTION
**Pull request by thomaslee1911**
This refactors out the geocoding logic from parse_csv, including viewport-biasing base on the default viewport parameter. @kyle-rosenbluth @sanjayss34 @sandlerben

### CODE REVIEW

File: app/utils.py [hunk 1]
```diff
@@ -29,3 +31,16 @@ def parse_phone_number(phone_number):
         stripped = '1' + stripped
     stripped = '+' + stripped
     return stripped
+
+
+# Viewport-biased geocoding using Google API
+# Returns a tuple of (latitude, longitude), (None, None) if geocoding fails
+def geocode(address):
+    url = "https://maps.googleapis.com/maps/api/geocode/json"
+    payload = {'address': address, 'bounds': config['default'].VIEWPORT}
```
Comment (sandlerben) [inline app/utils.py]: This isn't how you should get things from config. You should get stuff from app.config

File: app/utils.py [hunk 2]
```diff
@@ -29,3 +31,16 @@ def parse_phone_number(phone_number):
...
+    r = requests.get(url, params=payload)
+    if r.json()['status'] is 'ZERO_RESULTS' or len(r.json()['results']) is 0:
+        return (None, None)
```
Comment (sandlerben) [inline app/utils.py]: Nit: don't include redundant parenthesis around tuples

File: app/utils.py [hunk 3]
```diff
@@ -29,3 +31,16 @@ def parse_phone_number(phone_number):
...
+    else:
+        coords = r.json()['results'][0]['geometry']['location']
+        return (coords['lat'], coords['lng'])
```
Comment (sandlerben) [inline app/utils.py]: same as above

**Comment by sandlerben** [comment]
Had a few small comments, besides that lgtm

File: app/utils.py (hunk 2 above)
Comment (thomaslee1911) [inline app/utils.py]: Got it

File: app/utils.py (hunk 3 above)
Comment (thomaslee1911) [inline app/utils.py]: Ok

**Comment by thomaslee1911** [comment]
@kyle-rosenbluth it is merged.

**Pull request merged**